src/components/Forms/DonationForm/index.js

```diff
@@ -0,0 +1,441 @@
+                            DE74ZZZZ09671218105601
+                          </p>
+                        </Overlay>
+                        <Field
+                          name="privacy"
+                          label={
+                            <>
+                              Ich habe die{' '}
+                              <a
+                                href="/datenschutz/"
+                                target="_blank"
+                                className={s.link}
+                              >
+                                Datenschutzbedingungen
+                              </a>{' '}
+                              zur Kenntnis genommen.
+                            </>
+                          }
+                          type="checkbox"
+                          component={Checkbox}
+                        ></Field>
+                      </FormSection>
+
+                      <PrimarySecondaryButtonContainer>
+                        <InlineButton
+                          onClick={() => {
+                            setIsRecurring(false);
+                            setEnteredAmount(false);
+                          }}
+                        >
+                          Zurück
+                        </InlineButton>
+                        <Button type="submit" size="MEDIUM">
+                          Weiter
+                        </Button>
+                      </PrimarySecondaryButtonContainer>
+                    </div>
+                  )}
+                </form>
+              </FormWrapper>
+            );
+          }}
+        ></Form>
+      )}
+
+      {!hasDonated && enteredPaymentInfo && !donationError && (
+        <div>
+          <h3>Bitte überprüfe deine Daten</h3>
+
+          <p>
+            Name:{' '}
+            <span className={s.info}>
+              {donationInfo.donation.firstName} {donationInfo.donation.lastName}
+            </span>
+          </p>
+          <p>
+            IBAN: <span className={s.info}>{donationInfo.donation.iban}</span>
+          </p>
+          <p>
+            Mit dem Klick auf "Jetzt spenden" bestätigst du, dass du{' '}
+            <span className={s.info}>
+              {isRecurring ? 'monatlich' : 'einmalig'}{' '}
+              {donationInfo.donation.amount} €
+            </span>{' '}
+            an die Expedition spenden möchtest.
+          </p>
+
+          <PrimarySecondaryButtonContainer>
+            <InlineButton
+              onClick={() => {
+                setEnteredPaymentInfo(false);
+              }}
+            >
+              Zurück
+            </InlineButton>
+            <CTAButton
+              onClick={() => {
+                updateUser(donationInfo);
+              }}
+              size="MEDIUM"
+            >
+              Jetzt spenden
+            </CTAButton>
+          </PrimarySecondaryButtonContainer>
+        </div>
+      )}
+
+      {hasDonated && !donationError && (
```

Comment:
Does it maybe make sense here to use the FinallyMessage component? Maybe not, since the whole form is inside of a container anyway.